src/renderStream.ts

```diff
@@ -0,0 +1,43 @@
+import * as fs from 'fs';
+import * as util from 'util';
+
+import {renderSync} from './render';
+import {StyleConfig} from './types';
+import {validateRenderData} from './validate';
+
+const readAsync = util.promisify(fs.readFile);
```

Comment:
You could also `import * as fs from 'fs/promises'`

Reply:
Hmm, I tried this and it wasn't happy about getting `0` for the file descriptor

Reply:
https://humanwhocodes.com/snippets/2019/05/nodejs-read-stream-promise/ uhg node